src/containers/Tenant/Acl/Acl.tsx

```diff
@@ -80,14 +67,34 @@ export const Acl = ({path}: {path: string}) => {
             return null;
         }
 
-        return (
-            <ResizeableDataTable
-                columnsWidthLSKey={ACL_COLUMNS_WIDTH_LS_KEY}
-                columns={columns}
-                data={acl}
-                settings={TABLE_SETTINGS}
-            />
-        );
+        const normalizedAcl = normalizeAcl(acl);
+
+        const items = normalizedAcl.map(({Subject, ...data}) => {
+            return {
+                label: Subject,
+                items: aclParams
+                    .map((key) => {
+                        const value = data[key];
+                        if (!value) {
+                            return undefined;
+                        }
```

Comment:
It looks like `Access` is alway an array. 

![Screenshot 2024-07-02 at 11 45 07](https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/171b0f08-fc01-4d27-a211-79575cdb65e9)

Database: https://nda.ya.ru/t/aTbam5AN76jBrf